sonic-net/sonic-buildimage
# CMIS management feature enablement to MSFT HWSKU

#### Why I did it
Add CMIS management feature enablement to MSFT HWSKU

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
add feature variable to new file under MSFT HWSKU folder with value 1

#### How to verify it
install image on a swtich and check state machine and "show interface status" command output (Type column)



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
